rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -293,6 +293,8 @@ http {
     http2_max_field_size            {{ $cfg.HTTP2MaxFieldSize }};
     http2_max_header_size           {{ $cfg.HTTP2MaxHeaderSize }};
     http2_max_requests              {{ $cfg.HTTP2MaxRequests }};
+    large_client_header_buffers     {{ $cfg.LargeClientHeaderBuffers }};
```

Comment:
We have downgraded nginx to v1.19, so would hold this as well

Reply:
This option is already exists [here](https://github.com/kubernetes/ingress-nginx/blob/066a7d25b6bf8cdc06f5b5db4ee35241ee5163f5/rootfs/etc/nginx/template/nginx.tmpl#L289)

Reply:
Are you going to cover:
> nginx: [warn] the "http2_max_requests" directive is obsolete, use the "keepalive_requests" directive instead in /tmp/nginx-cfg1718353640:148

If not, I can make a pr for this...

Reply:
@jsoref    Please submit PR for the deprecated directives. Please double-check https://github.com/kubernetes/ingress-nginx/issues/7261 if the obsolete directives mentioned here are also taken care of